autoshop/login_signup/migrations/0003_auto_20170802_0621.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.3 on 2017-08-02 06:21
```

Comment:
Remove this migration. Generate a single migration as this will be our first database change.